docs/source/learn/core_notebooks/pymc_aesara.ipynb

```diff
@@ -0,0 +1,1960 @@
+{
```

Comment:
<p>Perhaps emphasize in text that we are just creating RandomVariables like we saw before, but now registering them in a PyMC model.</p><p> </p><p>And then it follows naturally to mention <code>model.basic_RVs</code> like you did </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='73'/>

Reply:
+1